tests/de/gurkenlabs/litiengine/environment/MapObjectLoaderTests.java

```diff
@@ -393,4 +393,16 @@ public int getBar() {
       return this.bar;
     }
   }
+
+  @Test
+  public void testMapObjectType(){
+
+    String mapObject1 = null;
+    String mapObject2 = "";
+
+    assertEquals(null, MapObjectType.get(mapObject1));
```

Comment:
Use `assertNull(...)` here and make sure that all instances of this are fixed in the PR.